### PR TITLE
fix ^M line endings

### PR DESCRIPTION
**Pull request by konstruktoid**
This PR looks big, but is actually just line ending conversions, inspired by #6200 (comment).
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6229/files?diff=unified&w=1

- Fix ^M line endings, https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings

The files still includes things like `audience:itpro^Mmanager:`, blank space endings and various Markdown linting errors.

Inconsistent configuration when editing or writing Markdown files?

```
windows-itpro-docs$ find ./ -name '*.md' | wc -l
4587
windows-itpro-docs]$ git status | grep -c modified
268
```

Signed-off-by: Thomas Sjögren <konstruktoid@users.noreply.github.com>

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
Please note: This PR does not contain any textual changes. 
There are only whitespace changes.  Use this view to confirm:
- [X] **Hite whitespace changes**:  https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6229/files?diff=unified&w=1

**Comment by illfated** [comment]
If you like, you can also verify this via the Rich diff view, but that feature only turns on for each individual file. I have take a screenshot to visualize it for you:

image

Note: I reduced my browser zoom to 67% to include more files in the list in the image above.

**Comment by illfated** [comment]
My main point in my previous 2 comments is that the only review needed is to verify that there are only whitespace changes and that they are useful ones (as stated in the main PR comment).

Any other form of text review only leads to reviewing seemingly random document pages without basing it on external feedback.

**Comment by illfated** [comment]
Sorry, I might have been too loud on the subject, taking the whitespace changes for granted. I will hide those comments (to keep them available for viewing, when required).

@konstruktoid : I could be mistaken, but it looks like GitHub indicates that some of those ^M line endings are still there. Can you confirm this?

image

File: browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
+
+
+### Supported values
+
+
+|                Group Policy                 | MDM | Registry |               Description               |                 Most restricted                  |
+|---------------------------------------------|:---:|:--------:|-----------------------------------------|:------------------------------------------------:|
+| Disabled or not configured<br>**(default)** |  0  |    0     |        No data collected or sent        | ![Most restricted value](../images/check-gn.png) |
+|                   Enabled                   |  1  |    1     |       Send intranet history only        |                                                  |
+|                   Enabled                   |  2  |    2     |       Send Internet history only        |                                                  |
```
Comment (JohanFreelancer9) [inline browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md]:
```suggestion
|                   Enabled                   |  2  |    2     |       Send internet history only        |                                                  |
```

File: browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md [hunk 2]
```diff
@@ -1,65 +1,65 @@
...
+|                   Enabled                   |  3  |    3     | Send both intranet and Internet history |                                                  |
```
Comment (JohanFreelancer9) [inline browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md]:
```suggestion
|                   Enabled                   |  3  |    3     | Send both intranet and internet history |                                                  |
```

File: browsers/edge/includes/configure-edge-kiosk-reset-idle-timeout-include.md [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
+
+
+### Related policies
+
+[Configure kiosk mode](../available-policies.md#configure-kiosk-mode): [!INCLUDE [configure-kiosk-mode-shortdesc](../shortdesc/configure-kiosk-mode-shortdesc.md)] 
+
+
+
+### Related topics
+[Deploy Microsoft Edge kiosk mode](../microsoft-edge-kiosk-mode-deploy.md): Microsoft Edge kiosk mode works with assigned access to allow IT administrators, to create a tailored browsing experience designed for kiosk devices. In this deployment guidance, you learn about the different Microsoft Edge kiosk mode types to help you determine what configuration is best suited for your kiosk device.  You also learn about the other group policies to help you enhance the how to set up your Microsoft Edge kiosk mode experience.
```
Comment (JohanFreelancer9) [inline browsers/edge/includes/configure-edge-kiosk-reset-idle-timeout-include.md]:
```suggestion
[Deploy Microsoft Edge kiosk mode](../microsoft-edge-kiosk-mode-deploy.md): Microsoft Edge kiosk mode works with assigned access to allow IT administrators to create a tailored browsing experience designed for kiosk devices. In this deployment guidance, you learn about the different Microsoft Edge kiosk mode types to help you determine what configuration is best suited for your kiosk device.  You also learn about the other group policies to help you enhance the how to set up your Microsoft Edge kiosk mode experience.
```

File: browsers/edge/includes/configure-microsoft-edge-kiosk-mode-include.md [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
+- **Path:** HKLM\Software\Policies\Microsoft\MicrosoftEdge\KioskMode
+- **Value name:** ConfigureKioskMode
+- **Value type:** REG_SZ
+
+### Related policies
+[Configure kiosk reset after idle timeout](../available-policies.md#configure-kiosk-reset-after-idle-timeout): [!INCLUDE [configure-kiosk-reset-after-idle-timeout-shortdesc](../shortdesc/configure-kiosk-reset-after-idle-timeout-shortdesc.md)] 
+
+
+### Related topics
+[Deploy Microsoft Edge kiosk mode](../microsoft-edge-kiosk-mode-deploy.md): Microsoft Edge kiosk mode works with assigned access to allow IT administrators, to create a tailored browsing experience designed for kiosk devices. In this deployment guidance, you learn about the different Microsoft Edge kiosk mode types to help you determine what configuration is best suited for your kiosk device.  You also learn about the other group policies to help you enhance the how to set up your Microsoft Edge kiosk mode experience.
```
Comment (JohanFreelancer9) [inline browsers/edge/includes/configure-microsoft-edge-kiosk-mode-include.md]:
```suggestion
[Deploy Microsoft Edge kiosk mode](../microsoft-edge-kiosk-mode-deploy.md): Microsoft Edge kiosk mode works with assigned access to allow IT administrators to create a tailored browsing experience designed for kiosk devices. In this deployment guidance, you learn about the different Microsoft Edge kiosk mode types to help you determine what configuration is best suited for your kiosk device.  You also learn about the other group policies to help you enhance the how to set up your Microsoft Edge kiosk mode experience.
```

Comment (JohanFreelancer9) [inline browsers/edge/includes/configure-microsoft-edge-kiosk-mode-include.md]:
```suggestion
[Deploy Microsoft Edge kiosk mode](../microsoft-edge-kiosk-mode-deploy.md): Microsoft Edge kiosk mode works with assigned access to allow IT administrators to create a tailored browsing experience designed for kiosk devices. In this deployment guidance, you learn about the different Microsoft Edge kiosk mode types to help you determine what configuration is best suited for your kiosk device.  You also learn about the other group policies to help you enhance the how to set up your Microsoft Edge kiosk mode experience.
```

File: browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md (hunk 1 above)
Comment (konstruktoid) [reply browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md]: Should these recommendation be added to the current PR or is it the base a new one?
This PR was supposed to be focused on linting and basic cleaning.

**Comment by konstruktoid** [comment]
> Sorry, I might have been too loud on the subject, taking the whitespace changes for granted. I will hide those comments (to keep them available for viewing, when required).
> 
> @konstruktoid : I could be mistaken, but it looks like GitHub indicates that some of those ^M line endings are still there. Can you confirm this?
> 
> image

https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6229/files#diff-5759daa8445550f308f68536814b52e3R6

File: browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md (hunk 1 above)
Comment (illfated) [reply browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md]: Good question. This is sort of what I initially tried to shout out (see my hidden messages above), but then I started doubting my own judgement capabilities.

Comment (konstruktoid) [reply browsers/edge/includes/configure-browser-telemetry-for-m365-analytics-include.md]: And then it's all downhill from there @illfated ;)

@JohanFreelancer9, my opinion is that we try to focus this PR to `^M` and similar things, and create a new PR, or a couple, regarding spelling, wording and structure.

**Comment by illfated** [comment]
> 
> We can't/should'nt remove all `^M` since they are not line endings, and perhaps correctly placed in the sentence, even if I doubt it.
>
In this very case, the text "**audience: itpro**" and "**manager: dansimp**" is meant to be on separate lines.
As a matter of fact, _all the keywords_ (text before the colon) in the metadata field are supposed to have their own separate line.

**Comment by konstruktoid** [comment]
> > We can't/should'nt remove all `^M` since they are not line endings, and perhaps correctly placed in the sentence, even if I doubt it.
> 
> In this very case, the text "**audience: itpro**" and "**manager: dansimp**" is meant to be on separate lines.
> As a matter of fact, _all the keywords_ (text before the colon) in the metadata field are supposed to have their own separate line.

Yeah, that's what I thought, but 268 files changed just by fixing some line endings, and it would require actually reading and linting every document if we want to catch issues like that.

I could do it, but in my opinion it's out of scope for this PR and we risk some major merge conflicts.

**Comment by illfated** [comment]
@JohanFreelancer9, @mypil : I hope you are aware that the copy review performed here in this PR is not based on any PR-specific changes, but rather the existing document content.

**Comment by illfated** [comment]
> Yeah, that's what I thought, but 268 files changed just by fixing some line endings, and it would require actually reading and linting every document if we want to catch issues like that.
> 
> I could do it, but in my opinion it's out of scope for this PR and we risk some major merge conflicts.

Fair enough. We will just have to wait for future PRs to fix those MS-DOS line endings.

**Comment by JohanFreelancer9** [comment]
@illfated  Yes I am fully aware of that but I cannot, in good conscience, approve the copy, even if it is existing copy if it contains errors or is not in accordance with the MS style guide.

**Comment by illfated** [comment]
Very well, fair enough. @konstruktoid : I recommend accepting the suggested changes.

**Comment by konstruktoid** [comment]
@illfated, commited.
@JohanFreelancer9, is the MS style guide available somewhere?

**Comment by JohanFreelancer9** [comment]
@konstruktoid Thank you for committing the changes. Here is a link to the Microsoft Style Guide: https://docs.microsoft.com/en-us/style-guide/welcome/

**Comment by illfated** [comment]
@konstruktoid : Please also perform a global replace for this ^M occurrence:
```diff
- audience: itpro^Mmanager: dansimp
+ audience: itpro
+ manager: dansimp
```

**Comment by konstruktoid** [comment]
@illfated, It has been done. And it's now 4,494 files to review. ¯\_(ツ)_/¯

**Comment by illfated** [comment]
Oh. Sorry. I was evidently too occupied with starting my regular job in addition to what I wrote. I did not intend to say "global replace" as in "the entire Universe of Microsoft Documentation", but rather that it should be applied to the already existing files in this PR. Sorry about that.

**Comment by konstruktoid** [comment]
"the entire Universe of Microsoft Documentation" in this repo was affected by this PR initially as well, but with all linting errors it amounted to quite a number of files.

There are no conflicts and it doesn't require alot of time to review since it's just a bunch of white spaces removed and document changes are easily spotted.

**Comment by illfated** [comment]
Very well, fair enough. I just hope the MS Docs team members and managers will react positively to this change.

**Comment by konstruktoid** [comment]
Another option for the MS Docs team members is to manually edit all the files.

**Comment by illfated** [comment]
Fair point. I often see contributors make PRs containing only 1 file, but very often with that change incorporated due to automatic correction.

**Comment by illfated** [comment]
BTW, besides the repository content taking up so much space in this PR, could you check to see why some files still have got the ^M character in the metadata section, as we discussed before?
They show up even when applying the Hide whitespace changes feature:
- [X] [Hide whitespace changes]:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6229/files?diff=unified&w=1

**Comment by konstruktoid** [comment]
I can't find any files with `^M`, do you have any example?

```
$ git grep -c '^M' '*.md'
$ 
```

**Comment by illfated** [comment]
Sorry about that, my bad. I scrolled too quickly through the files with "Hide whitespace changes" turned on and thought that I saw them in the green (updated) lines. Turns out they are only in the red (previous version) lines. You are correct. That was a complete misread on my part. Not to worry, it looks good to me.

**Comment by konstruktoid** [comment]
A bunch of conflicts that must be resolved, but too complex in the web editor, so I'm just going to wait for a response if this is going to get merged if the conflicts (or reverting of the affected files) is fixed.

**Comment by illfated** [comment]
Yeah, well, I sort of saw this coming, although without knowing how soon. For cases like this, I would use local Git tools (in my case, Git Extensions) to handle all the PR files and resolve the merge conflicts via kdiff and the rest of the tools in the GitExtensions package. You may of course choose whichever solution you find works for you, this is mostly me sharing my thoughts on the matter.

**Comment by konstruktoid** [comment]
I normally just use the git cli with vim, and (without checking) I don't think these are very complex conflicts, just a lot.

Or, as I wrote earlier, the MS team could just decide to merge conflicting files from `public` and still get a majority of the fixes.

**Comment by illfated** [comment]
Fair enough, I am glad you are using any git tools to handle the files.
I will wait for feedback from the MS Docs team regarding this.

**Comment by mypil** [comment]
@Dansimp - PR is ready for final review.  

Thank you.

cc: @kenwith

**Comment by e0i** [comment]
Hello, @konstruktoid There has been a confirmation that the review is going to proceed upon resolving merge conflicts. Can you follow?

**Comment by konstruktoid** [comment]
Hi again @e0i.  I'm working on it but since the amount of conflicting files is quite high, I will most likely be finished tomorrow.

**Comment by konstruktoid** [comment]
rebased @e0i, https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6229/files?diff=unified&w=1

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@konstruktoid It is difficult to catch up with the pace of regular edits as this PR modifies about 2,000+ files in the repository, thus merge conflicts keep resurfacing. Would you be able to resolve merge conflicts again? I'll try my best to have a merge strike on time. Thank you.

**Comment by konstruktoid** [comment]
Branch deleted.

**Pull request closed without merging**